<!-- -->
<launch>

  <arg name="input_odom_topic" default="/localization/gps/odom" />
  <arg name="lidar2imu_rotation" 
      default="[0.9982, -0.04619, -0.038339, 0.046762, 0.99881, 0.014155, 0.03764, -0.015922, 0.99916]" />
  <arg name="lidar2imu_translation" 
      default="[0.93, 0.05, 0.95]" />  

  <node pkg="tf4hmi" type="tf4hmi" name="tf4hmi" output="screen">
    <param name="input_odom_topic" value="$(arg input_odom_topic)"  />
    <param name="lidar2imu_rotation" value="$(arg lidar2imu_rotation)"  />
    <param name="lidar2imu_translation" value="$(arg lidar2imu_translation)"  />
  </node>


</launch>
